Documentation/trying-out-rkt.md

```diff
@@ -16,6 +16,8 @@ cd rkt-v1.8.0
 ./rkt help
 ```
 
+Note that in order to fully utilize rkt there is a couple of additional steps to be performed. Therefore we highly recommend installing rkt using your local package manager. If rkt is not available for your Linux distribution please consider to try rkt using Vagrant as described below.
```

Comment:
If we mention apt on Debian or dnf on Fedora, we could as well just link to the documentation [Installing rkt on popular Linux distributions](https://github.com/coreos/rkt/blob/master/Documentation/distributions.md).